presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcConfig.java

```diff
@@ -61,4 +63,28 @@ public BaseJdbcConfig setConnectionPassword(String connectionPassword)
         this.connectionPassword = connectionPassword;
         return this;
     }
+
+    public String getUserCredentialName()
```

Comment:
Add `@Nullable`

Reply:
Should we also add `@Nullable` to `getConnectionUser()` and `getConnectionPassword()`?

Reply:
Yes, please.

Reply:
Remember to do it in separate commit.